student-source/students/accessing-codio/faq.rst

```diff
@@ -101,13 +101,16 @@ Accessing assignments when using an LMS platform
 ------------------------------------------------
 
 If LTI is enabled for a class, new assignments can only be started via the LMS platform. Any assignments already started can be accessed by logging into your Codio account.
+Your teacher/instructor can set up a link so if you try to start an assignment you can be given a link to click that will take you to your LMS course.
 
 - Assignments that have not been started will show with 'ready to go' in the Codio course
 - Assignments that have been started previously will show with 'In Progress' in the Codio course
 
     .. figure:: /img/student-dash.png
        :alt: Students View
        
+.. Note:: Teachers in your organization can hide assignments that are not been started from your dashboard so you may only see those you have already started.
```

Comment:
are->have

Note:: Teachers in your organization can hide assignments that have not been started from your dashboard so you may only see those you have already started.